docs/decisions/0009-versioning-for-phdi-lib.md

```diff
@@ -0,0 +1,66 @@
+    * Impossible for users to tell at a glance whether a given version upgrade will break their existing functionality
+3. **Sequential Number Versioning**
+- Defined as versioning that begins with "1" and increments the count with each successive software push
+- Examples: Not really any that work out in the wild
+- Pros:
+  * Largely the same as date-of-release versioning without the drawback of an imposed / artificial schedule
+- Cons:
+  * Also susceptible to version verbosity, as with date versioning
+  * The optics of a massively large number for a version number (i.e. 427) could reduce confidence in our ability to make correct software the first time
+  * Creates the temptation to not increment the version number to avoid the above problem, which then eliminates consistency as well
+4. **Semantic Versioning**
+- Defined as versioning of the form A.B.C. _A_ corresponds to the "major release" number and documents the addition, alteration, or removal of features sufficient to break the existing API. _B_ corresponds to the "minor release" number and documents the addition or removal of features which _do not_ break the existing API. _C_ corresponds to the "patch version" and documents the bug fixes that have been applied to the software.
+- Examples: All of Google's internal and external software platforms; the R kernel
+- Pros:
+  * Achieves transparency by clearly delineating when an aspect of the software changes, as well as showing what that aspect was
+  * Achieves consistency by separating out breaking vs non-breaking API changes
+  * Achieves interpretability by adding a "patch version" to MAJOR.MINOR versioning schemes, which conveys whether particular bugs have been fixed or not
+  * Well-documented, formally used process that reduces ambiguity on the developer side, to the extent that many major software developers around the world use it
+  * Optics around version numbers make intuitive sense to users; in other words, they should "feel natural" in a world where things like e.g. operating system numbers are of the form 14.12.21, which allows developers to create lots of changes and alter the version number without creating the feeling of an unstably changing software platform
+- Cons:
+  * May require careful consideration and discussion around whether a change has actually "broken" the existing API
+  * Bug fix / patch number can sometimes be treated as a "catch all" bucket and wind up spiraling to large numbers like sequential number versioning
+
+## Decision Outcome
+
+We are adopting Semantic Versioning to document the PHDI building blocks library's development. It is robust, standardized, and well-documented with precise protocols for when things change and what those changes should be. It will achieve all three major decision drivers and provide additional clarity for both users and developers.
```

Comment:
> It will achieve all three major decision drivers

This sentence needs to be updated now that we've gone down to two decision drivers.